lesson01/assignment/index.html

```diff
@@ -1,25 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Assignment 1 - Part 2 Example</title>
-  <script src="scripts.js"></script>
-  <link rel="stylesheet" href="style.css">
-</head>
-
-<body>
-  <main>
-    <h1>Using ES6+ Methods in DOM Interactions</h1>
-    <form id="example-form">
-      <label>Enter your favorite number:
-        <input type="text" class="input">
-      </label>
-      <input type="submit" value="See Demo!">
-      <p>The total of my array elements summed with your favorite number is: <span class="output"></span></p>
-    </form>
-  </main>
-</body>
-
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <title>Assignment 1 - Part 2 Example</title>
+      <script src="scripts.js"></script>
+      <link rel="stylesheet" href="style.css">
+   </head>
+   <body>
+      <main>
+         <h1>Using ES6+ Methods in DOM Interactions</h1>
+         <form id="form1">
+            <label>Enter your favorite number :
+            <input type="text"  class="input">
+            </label>
+            <input type="submit"  value="Function 1 !">
+            <p>Map Function, we are going to add the favorite number to the array [0, 1, 2, 3, 5, 8] <span class="output" style="color: green;"></span></p>
```

Comment:
Inline styles are no longer considered good practice. Since you want all output text to have green text, move the rule into SCSS `span.output {color: green;}`